bucket/qtox-portable.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": "1.16.3",
+    "description": "Chat, voice, video, and file transfer IM client using the encrypted peer-to-peer Tox protocol.",
+    "homepage": "https://qtox.github.io/",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/qTox/qTox/releases/download/v1.16.3/setup-qtox-x86_64-release.exe#/dl.7z",
+            "hash": "e4a36d89739768afedf539cd049bfc7eb19022a2f9c40e52b50081604b0ab027"
+        },
+        "32bit": {
+            "url": "https://github.com/qTox/qTox/releases/download/v1.16.3/setup-qtox-i686-release.exe#/dl.7z",
+            "hash": "d4a50995124b6663c4aa7398520f023e7a76cba9a179371af5c9e45396b3c551"
+        }
+    },
+    "extract_dir": "bin",
+    "installer": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\\qtox.ini\")) {",
+            "    if (Test-Path \"$env:AppData\\Tox\" -PathType Container) {",
+            "        Copy-Item \"$env:AppData\\Tox\\*\" \"$dir\" -Include '*.ini', '*.db', '*.tox'",
+            "    }",
+            "    if (Test-Path \"$dir\\qtox.ini\") {",
+            "        ((Get-Content \"$dir\\qtox.ini\") -replace 'makeToxPortable=false', 'makeToxPortable=true') | Set-Content \"$dir\\qtox.ini\" -Encoding ASCII",
+            "    }",
+            "    else { Set-Content \"$dir\\qtox.ini\" @('[Advanced]', 'makeToxPortable=true') -Encoding ASCII -Force }",
+            "}",
+            "else { Copy-Item \"$persist_dir\\*\" \"$dir\" -Exclude 'qtox.ini' }"
+        ]
+    },
+    "uninstaller": {
+        "script": "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include '*.ini', '*.db', '*.tox' -Force"
+    },
+    "bin": "qtox.exe",
+    "shortcuts": [
+        [
+            "qtox.exe",
+            "qTox"
+        ]
+    ],
+    "persist": "avatars",
+    "checkver": {
+        "github": "https://github.com/qTox/qTox/"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/qTox/qTox/releases/download/v$version/setup-qtox-x86_64-release.exe#/dl.7z"
```

Comment:
You most likely need to remove $PLUGINSDIR

Reply:
There is no $PLUGINSDIR or uninstall.